src/gpu/intel/jit/conv/config.cpp

```diff
@@ -1624,16 +1624,16 @@ class mn_walker_t {
 
 walk_order_t compute_walk_order(const conv_config_t &cfg) {
     auto &prb = cfg.prb();
-    int tg_size = 1;
+    int thr_size = 1;
```

Comment:
Should be `tg_size`? as this is the threadgroup size.

Reply:
Done.